4.classify-single-cell-phenotypes/4b.derive-classification-profiles/4b.derive-classification-profiles.ipynb

```diff
@@ -0,0 +1,607 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    cell_health_plate_classifications = pathlib.Path(
</pre><p>If this data is not confidential or not too large, consider allowing open access</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/WayScience/cell-health-data/pull/20/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
The plan is to upload this data to Zenodo (open access data storage) in the near future! I will include the info for accessing this data as soon as possible.